website/docs/zh/guide/basic/output-format.mdx

```diff
@@ -45,7 +45,7 @@ Rslib 支持四种 JavaScript 文件的输出格式：`esm`、`cjs`、`umd` 和
   - 一些 Node.js 工具和库仍然对 ESM 有限制或不支持，需要解决方法或额外配置。
   - 你必须在导入路径中显式包含文件扩展名，这可能会很麻烦，尤其是在使用 TypeScript 或其他转译语言时。
 
-#### 发布 [ESM 和 CJS（双重）](https://antfu.me/posts/publish-esm-and-cjs#compatibility) 包
+#### 发布 [ESM 和 CJS (双重)](https://antfu.me/posts/publish-esm-and-cjs#compatibility) 包
```

Comment:
Full-width symbols should be used in Chinese.

Reply:
![image](https://github.com/user-attachments/assets/9a637f3b-33b0-4e44-9da5-3ae981cf08c2)

The Chinese brackets are a bit wide and don't look pretty in the title.

Reply:
Since this is not a mistake, and it makes sense, I am open to it.